frontend/src/lib/api.ts

```diff
@@ -256,6 +265,28 @@ const api = {
         },
     },
 
+    activity: {
+        list(activityLogProps: ActivityLogProps, page: number = 1): Promise<CountedPaginatedResponse> {
+            const urlForScope: {
+                [key in ActivityScope]: (props: ActivityLogProps) => string
+            } = {
+                [ActivityScope.FEATURE_FLAG]: (props) => {
+                    return props.id
+                        ? `/api/projects/@current/feature_flags/${props.id}/activity`
+                        : `/api/projects/@current/feature_flags/activity`
+                },
+                [ActivityScope.PERSON]: (props) => {
+                    return props.id ? `/api/person/${props.id}/activity` : `/api/person/activity`
```

Comment:
Ideally we should use the other api builder methods to gradually create the URL, instead of hardcoding it here. Happy to yolo if it's weirdly verbose or too complicated...